ipfsApi/client.py

```diff
@@ -201,16 +202,21 @@ def add(self, files, recursive=False, **kwargs):
         return self._add.request(self._client, (), files,
                                  recursive=recursive, **kwargs)
 
-    def get(self, multihash, **kwargs):
+    def get(self, multihash, output_path='.', **kwargs):
         """Downloads a file, or directory of files from IPFS.
 
-        Files are placed in the current working directory.
+        Files are placed in the current working directory if
+        output_path is not specified. Otherwise, they are
+        moved to output_path.
 
         Keyword arguments:
         multihash -- unique checksum used to identify IPFS resources
+        output_path -- where to put the downloaded file
         kwargs -- additional named arguments
         """
-        return self._get.request(self._client, multihash, **kwargs)
+        self._get.request(self._client, multihash, **kwargs)
+        if output_path is not '.':
+            shutil.move(multihash, output_path)
```

Comment:
Good question. I hadn't considered file system boundaries or writability, though I should mention that since the `get` function always downloads to the current working directory, I'd sincerely hope that it's writable. 
As I'm sure you can tell, I'm no Python pro. I know just enough that I was able to patch up the repo. I'm happy to help maintain this project, but I'm definitely still learning best-practices (and some features) in the language.
The other approach that I can see is to push this logic down into the `DownloadCommand` or maybe further into the `http` module. This could ensure that the file is never written to the CWD unless the user didn't specify another path. However, I'm not certain whether that logic is unique to the `get` function. I'll have to check on that.

Do you see any other alternatives?
Also, do you think that the module's current assumptions about the ability to write to the CWD represent a systemic design flaw?

Thanks so much for these code reviews, by the way.